README.md

```diff
@@ -3,6 +3,7 @@
 - [Development](#development)
     - [Gotchas](#gotchas)
 - [Manual testing](#manual-testing)
+- [Automated testing](#automated-testing)
```

Comment:
@BriDavidson do you think we should change this to "E2E testing" or something? Lest it gets conflated with unit testing

Reply:
I was thinking this as well but wasn't sure how detailed we wanted to get with this. I removed this.